src/BookingConflictConditionFactory.php

```diff
@@ -126,27 +169,27 @@ public function make($config = null)
             );
         }
 
+        // For each non-blocking status, AND the condition to a "NOT EQUAL TO" sub-condition for that status
+        // This makes the condition disregard bookings with these statuses
+        foreach ($this->_getNonBlockingStatuses() as $nonBlockingStatus) {
+            $b->and(
+                $condition,
+                $b->not(
+                    $b->eq(
```

Comment:
Why use a loop and an `eq` comparison, when you could be using an `in` comparison, which would avoid having to create several similar conditions?

Reply:
Good point.